dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/Parameters/ParamsREPC.mo

```diff
@@ -16,8 +16,8 @@ within Dynawo.Electrical.Controls.WECC.Parameters;
 record ParamsREPC
   parameter Boolean FreqFlag "Governor response disable (0) or enable (1)" annotation(
   Dialog(tab="Plant Control"));
-  parameter Boolean RefFlag "Plant level reactive power (0) or voltage control (1)" annotation(
-  Dialog(tab="Plant Control"));
+//  parameter Boolean RefFlag "Plant level reactive power (0) or voltage control (1)" annotation(
+//  Dialog(tab="Plant Control"));
```

Comment:
Please uncomment these lines and convert RefFlag into an Integer parameter, as you have done in PVVoltageSourceAREPCc.mo.